mappings/net/minecraft/scoreboard/Scoreboard.mapping

```diff
@@ -6,6 +6,13 @@ CLASS net/minecraft/class_269 net/minecraft/scoreboard/Scoreboard
 	FIELD field_1430 displaySlotNames [Ljava/lang/String;
 	FIELD field_1431 playerObjectives Ljava/util/Map;
 	FIELD field_1432 objectiveSlots [Lnet/minecraft/class_266;
+	FIELD field_31886 LIST_DISPLAY_SLOT_ID I
+	FIELD field_31887 SIDEBAR_DISPLAY_SLOT_ID I
+	FIELD field_31888 BELOW_NAME_DISPLAY_SLOT_ID I
+	FIELD field_31889 MIN_SIDEBAR_TEAM_DISPLAY_SLOT_ID I
+	FIELD field_31890 MAX_SIDEBAR_TEAM_DISPLAY_SLOT_ID I
```

Comment:
Are these used as method parameters or return values? If yes consider unpicking them

Reply:
No they're used in switches